openslides/core/static/js/core/base.js

```diff
@@ -39,40 +39,41 @@ angular.module('OpenSlidesApp.core', [
     'DS',
     '$rootScope',
     function (DS, $rootScope) {
-        var url = location.origin + "/sockjs";
-        var socket;
+        var socket = null;
+        var recInterval = null;
+        $rootScope.connected = true;
+
+        var newConnect = function () {
+            socket = new SockJS(location.origin + "/sockjs");
+            clearInterval(recInterval);
+            socket.onopen = function () {
+                $rootScope.connected = true;
+            };
+            socket.onclose = function () {
+                $rootScope.connected = false;
```

Comment:
1 second delay (in css animation) is not the best solution (because in some large list views a reload need more than 1 second.
Currently, this "hide red bar on startup" feature is not really important - I like it even if the blue loading bar overrides the red bar. So "want fix" this red bar issue.

Please merge.